### PR TITLE
close issue #3 explain and ask for config on first use

### PR DESCRIPTION
**Pull request by marian-vignau**
#3 on first run, program ask for configuration

### CODE REVIEW

File: recordium/main.py [hunk 1]
```diff
@@ -209,12 +222,17 @@ def set_message_number(self):
...
-        self.setQuitOnLastWindowClosed(False)  # so app is not closed when closing other windows
-        self.storage = storage.Storage()
-        self.systray = SysTray(self, version)
-        self.messages_getter = network.MessagesGetter(
-            self._new_messages, self.storage.get_last_element_id)
-        self.messages_getter.go()
+        if not config.get(config.BOT_AUTH_TOKEN):
+            self._temp_cw = ConfigWidget(explain=True)
+            self._temp_cw.exec_()
+        if config.get(config.BOT_AUTH_TOKEN):
```
Comment (facundobatista) [inline recordium/main.py]: No matter if the user configured something or not, the app should start as usual, so remove this "if" and let the app just start

File: recordium/main.py [hunk 2]
```diff
@@ -209,12 +222,17 @@ def set_message_number(self):
...
+            """Start application network and systray menu & icon"""
```
Comment (facundobatista) [inline recordium/main.py]: This string sould be a comment

**Review by facundobatista — CHANGES_REQUESTED** [review]
A couple of details...

**Comment by marian-vignau** [comment]
done... see it pls

**Comment by facundobatista** [comment]
You did a change for good and another for bad :/

- For good: it's great that everytime you go to the config, and the token is not configured, you see the message

- For bad: you changed something that was GREAT in the previous version of your code: that when you start the program and the token is not configured, the config window is opened to you.

So, please restore the wonderful thing you did before of opening the config dialog on start if the token wasn't there (note that when the user closes that config, no matter if it's well configured or not, the program just should continue ok).

**Comment by marian-vignau** [comment]
done

**Comment by facundobatista** [comment]
Wonderful, thanks!

**Pull request merged**